bridge/.gitignore

```diff
@@ -1,7 +1,12 @@
 infura.key
 mnemonic.key
 etherscan.key
-build
+#Build
+/out
+/build
+node_modules
```

Comment:
this is already ignored in the top-level .gitignore, right?

Reply:
true